src/elfpy/markets.py

```diff
@@ -370,7 +370,7 @@ def open_long(
     def close_long(
         self,
         wallet_address: int,
-        trade_amount: float,
+        trade_amount: float,  # in bonds
```

Comment:
No need to do it in this PR, but the desire to add the unit comments seems like a good enough argument to use the `types.Quantity` dataclass here, which has a unit attribute.

Reply:
@wakamex could you add a TODO in the docstrings to remind us?